### DISCUSSION TITLE
Make hidden .folder visible again

### DISCUSSION THREAD

**Discussion by kurrrioo** [zeapo/Android-Password-Store#2833] [Q&A] [answered]
Greetings,
Plz help me to change the name of an invisible folder. The Android device has no root.
(Sorry if I have spoiled anything.)

**Comment by msfjarvis** [comment]
I don't understand the question, are you trying to rename a folder in your password store? APS does not use root either way so that's immaterial.

**Comment by kurrrioo** [comment, marked as answer]
I found a solution in the settings menu. Once turned on, I was able to remove the dot in the name of the hidden folder. There was another directory ".git", as well, which is supposed to be invisible to (most) users.

I enjoy your app. Next I will try to use the auto-fill option.

Thank you very much. I will consider to buy the devs a coffee as suggested on the f-droid platform.
